main/res/values/strings.xml

```diff
@@ -814,11 +814,13 @@
     <!-- map sources -->
     <string name="map_source_google_map">Google: Map</string>
     <string name="map_source_google_satellite">Google: Satellite</string>
+    <string name="map_source_osm_attribution_html"><![CDATA[© <a href="http://www.openstreetmap.org/" style="color: blue">OpenStreetMap</a> contributors]]></string>
```

Comment:
Do we need this as translatable string?

Reply:
I thought that maybe the link can lead to country-specific openstreetmap websites e.g. openstreetmap.de or openstreetmap.fr..

What do you think?

Reply:
See my comment below. The attribution is specific to a map source. In case of OpenStreetMap it must link to the copyright page.

Reply:
ok, will do it, see my reply there

Reply:
all done